sgx_main.c

```diff
@@ -119,7 +119,7 @@ static unsigned long sgx_get_unmapped_area(struct file *file,
 					   unsigned long pgoff,
 					   unsigned long flags)
 {
-	if (len < 2 * PAGE_SIZE || (len & (len - 1)) || flags & MAP_PRIVATE)
+	if (flags & MAP_PRIVATE)
```

Comment:
len < 2 * PAGE_SIZE is still needed because it ch3ecks the least ssize.
SEC page and TCS page.

Reply:
just a thought,
What if we map enclave starting from zero, but mmap() it from > 0.
and then untrusted pal maps from zero?
what happens?